.github/workflows/assign_project_to_issue.yml

```diff
@@ -0,0 +1,15 @@
+name: Auto Assign Issue to Project
+
+on:
+  issues:
+    types: [opened, transferred]
+
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.4.0
+        with:
+          project-url: https://github.com/orgs/AbsaOSS/projects/7
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
ADD_TO_PROJECT_PAT - I assume that this is an internal thing coming from Github itself? I couldn't find it in the repo's secrets

Reply:
I copied this form [atum-service](https://github.com/AbsaOSS/atum-service/blob/master/.github/workflows/assign_project_to_issue.yml), have to recall how I was setting this up there...